src/main/java/frc/team2412/robot/commands/autonomous/TwoBallAutoCommandLeft.java

```diff
@@ -0,0 +1,44 @@
+package frc.team2412.robot.commands.autonomous;
+
+import org.frcteam2910.common.control.SimplePathBuilder;
+import org.frcteam2910.common.control.Trajectory;
+import org.frcteam2910.common.math.Rotation2;
+import org.frcteam2910.common.math.Vector2;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.ParallelDeadlineGroup;
+import edu.wpi.first.wpilibj2.command.ScheduleCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.team2412.robot.commands.index.IndexShootCommand;
+import frc.team2412.robot.commands.intake.IntakeExtendCommand;
+import frc.team2412.robot.commands.intake.IntakeInCommand;
+import frc.team2412.robot.commands.shooter.ShooterTargetCommand;
+import frc.team2412.robot.subsystem.DrivebaseSubsystem;
+import frc.team2412.robot.subsystem.IndexSubsystem;
+import frc.team2412.robot.subsystem.IntakeSubsystem;
+import frc.team2412.robot.subsystem.ShooterSubsystem;
+import frc.team2412.robot.subsystem.ShooterVisionSubsystem;
+
+public class TwoBallAutoCommandLeft extends SequentialCommandGroup {
+    public TwoBallAutoCommandLeft(IndexSubsystem indexSubsystem, ShooterSubsystem shooterSubsystem,
+            ShooterVisionSubsystem shooterVisionSubsystem, DrivebaseSubsystem drivebaseSubsystem,
+            IntakeSubsystem intakeSubsystem) {
+        Trajectory robotPath = new Trajectory(
+                new SimplePathBuilder(new Vector2(337, 133), Rotation2.fromDegrees(-26.4))
+                        .lineTo(new Vector2(449, 84), Rotation2.fromDegrees(26.7))
```

Comment:
How far past the second ball does the robot go before stopping and shooting? (Does it make sure it goes far enough to pick the ball up?)
Is the angle of the robot at the end the best for shooting or the beginning of teleop?

Reply:
the robot is set to go essentially on top of the ball. The angle at the end has the turret facing toward the hub, and the robot facing away from it.